articles/how-to-convert-text-to-emoji-using-python/index.md

```diff
@@ -0,0 +1,155 @@
+
+### Dictionary
+Dictionaries stores data values in key pairs. A dictionary is a list of objects that can be rearranged without repeating its data values. You write the keys and values in dictionaries in curly brackets, it is an ordered list of data values. The key name refers to dictionary objects, which are presented in keys; value pairs.
+
+```Python
+details = { "brand": "Ford",
+           "model": "Mustang",
+           "year" : 2001}
+print(details.get("brand"))
+print(details.get("model"))
+print(details.get("year"))
+```
+
+In the above illustration, you can use the `.get()` method to get values stored in a dictionary.
+The  `.update()` methods will update the items from an argument. If the item does not exist, then it will add the item to the dictionary. Other methods you can use are:
+`.pop()` this method removes the item with the specific key name
+`.del()` removes the item with the specified key name
+`.clear()`method empties the dictionary
+Try more tricks using the  `dictionary.get()` method.
+Kindly try this yourself.
+To determine how many items a dictionary has, use the `print(len(details))`
+
+### Text to emoji converter
+You may believe that printing emojis would require a complex algorithm. However, let me assure you that this is not the case; writing the code to convert texts to emoji is straightforward. This is another excellent application of using a dictionary.
+The first step is to call the input function stored in a string. Next, you will call the `.split(“ ”)` method. This method goes through this string, and anywhere it finds a space, it uses it to separate this string into multiple words and return it as a list. Also, you define a dictionary for mapping special characters into emojis features.  you will then add key values of the pair and map it to a specific emoji character.
+You can press Ctrl, Command, and Space key on your keyboard to bring out the emoji box on your Mac System. On your Windows System, press and hold the Window button and either the  Fullstop (.) or semicolon (;) until you see the emoji picker appear.
+Finally, You will then loop through the string using the `.get()` method.
+See the example below.
+
+```Python
+message = input("> ")
+words = message.split( " ")
+emojis = {
+   ":)" : "😀",
+   ":(" : "😞",
+   "lol" : "😂",
+   "sick":"😨",
+   "happy": "😀",
+   "mermaid": "🧜‍"
+}
+outcome = " "
+for word in words:
+   outcome += emojis.get(word, word) + " "
+print(outcome)
+```
+
+you will use the `def` function to transform this code into a reusable function that we can use in various applications.
+
+```Python
+def  emoji_converter(message):
+ 	words = message.split( " ")
+emojis = {
+   ":)" : "😀",
+   ":(" : "😞",
+   "lol" : "😂",
+   "sick":"😨",
+   "happy": "😀",
+   "mermaid": "🧜‍"
+}
+outcome = " "
+for word in words:
+   outcome += emojis.get(word, word) + " "
+return output
+
+
+Message = input  (“>”)
+print(emoji_converter(message) )
+```
+
+Pretty easy, right?
+
+*Try this yourself*
+
```

Comment:
Given a sample text, display the output of the program. Let the users know what to expect from the code.

Reply:
Thanks. errors have been noted and changes implemented.